stubs/pyxdg/xdg/BaseDirectory.pyi

```diff
@@ -0,0 +1,19 @@
+from collections.abc import Iterator
+from os import PathLike
+from typing import AnyStr
+
+xdg_data_home: str
+xdg_data_dirs: list[str]
+xdg_config_home: str
+xdg_config_dirs: list[str]
+xdg_cache_home: str
+xdg_state_home: str
+
+def save_config_path(*resource: PathLike[AnyStr]) -> str: ...
```

Comment:
AnyStr is correct for these functions

Reply:
You got me double-checking and turns out it's not :-) They do joins from `os.environ` rather than `os.environb`, so they all need to be strings